### DISCUSSION TITLE
Feature Request - Ability to reorder container tabs

### DISCUSSION THREAD

**Discussion by Nitrousoxide** [Floorp-Projects/Floorp#616] [Feature Request]
Currently, container tabs cannot be reordered.  This means that any new containers you make will be in the order of creation, with no rhyme or reason to them besides that.  The ability to reorder the tabs in the settings page will be useful for the user if they create new ones.

<img width="735" alt="image" src="https://github.com/Floorp-Projects/Floorp/assets/1363931/28ac0858-40fb-4544-b0e7-80ac44c8ec87">

**Comment by ioc2e3** [comment]
Sorry, a question...

How to use container tab?
